src/main/java/aquarium/shop/order/DeliveryDetails.java

```diff
@@ -0,0 +1,34 @@
+package aquarium.shop.order;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "delivery_details")
+public class DeliveryDetails {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
We can try to increment ids per table not per all tables, I believe that SEQUENCE strategy can be used to acheve that:
https://stackoverflow.com/questions/62548985/hibernate-generates-one-id-for-two-tables-entities